### ISSUE TITLE
Question about implementation details

### ISSUE THREAD

**Issue by SonwYang** [bwconrad/decoder-denoising#1]
Hello, bro! Nice work! Can you tell me about the number of epoch in Denoising pretraining for decoder only?

**Comment by bwconrad** [comment]
According to the original paper, they pretrain for 100 epochs on ImageNet-21k.

**Comment by SonwYang** [comment]
Thank you! I have another question:What kind of learning rate schedule&nbsp;used in training model?



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Ben ***@***.***&gt;; 
发送时间: 2022年8月4日(星期四) 下午5:59
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [bwconrad/decoder-denoising] Question about implementation details (Issue #1)





 
According to the original paper, they pretrain for 100 epochs on ImageNet-21k.

**Comment by bwconrad** [comment]
They train following a cosine annealing schedule which is what is used in this repo as well.
